src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,47 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string,
+  description: string,
```

Comment:
Is required?

Reply:
What do You mean ? in case if there is no 'description' I set default property  '' (empty string) on line 14. That's why I didn't use 'isRequired'

Reply:
But typescript must know that it isn’t required props )

Reply:
Changed, and I hope correct)